src/views/Home/components/About/index.view.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from "react"
+import "./index.scss"
+
+const AboutView: React.FC = () => (
+  <>
+    <p>About</p>
```

Comment:
Let's make sure this is consistent among all view components the outside shell should be 
`<div className="<name>-view/container"`. This is my personal opinion but you should almost always use `<div>` for separating and writing text. It helps for styling since we most likely will be using [flex](https://www.w3schools.com/cssref/css3_pr_flex.asp) and it is much better designed for dividers.